dali/dfu/dfurun.cpp

```diff
@@ -1355,23 +1341,24 @@ class CDFUengine: public CInterface, implements IDFUengine
                         }
 
                         bool dirPerPart = false;
-                        //MORE: This could be combined with the code that gets defaultSprayParts
-                        if (isContainerized())
+                        IDFUfileSpec *dst = wu->queryUpdateDestination();
+                        StringBuffer clusterName;
+                        destination->getGroupName(0, clusterName);
+                        Owned<IPropertyTree> plane = getStoragePlane(clusterName);
+                        if (plane)
+                            dirPerPart = plane->getPropBool("@subDirPerFilePart", true);
```

Comment:
Probably, but we do not want to change these semantics in bare-metal - this change would alter that.
It means all new files (but not old) would have a new structure. If there mixed version environments that would be one way it would cause chaos.